resources/authgear/templates/en/web/authflowv2/enter_oob_otp.html

```diff
@@ -64,16 +64,8 @@ <h2 class="screen-description">
         )
       }}
 
-      {{ if $.CanRequestDeviceToken }}
-      <div class="mt-4 flex flex-row items-center gap-x-2.5">
-        <div class="checkbox">
-          <input form="main-form" id="device-token" type="checkbox" name="x_device_token" value="true">
-          <label class="checkbox__label" for="device-token">
-            <span class="checkbox__icon material-icons">check</span>
-          </label>
-        </div>
-        <label class="text-sm primary-txt" for="device-token">{{ template "create-device-token-label" }}</label>
-      </div>
+      {{ if $.DeviceTokenEnabled }}
+        {{ template "authflowv2/__device_token_checkbox.html" (dict "Form" "main-form" "Classname" "mt-[18px]") }}
```

Comment:
Reason to use mt-[18px] instead of mt-4?

Reply:
Oops sor, pointed to wrong element on figma to measure the margin